current/runtime/storage.md

```diff
@@ -56,38 +234,61 @@ decl_storage! {
 }
 ```
 
-## Verify First, Write Last
+### Genesis Config
+
+You can define
+[an optional `GenesisConfig`](https://substrate.dev/rustdocs/master/frame_support/macro.decl_storage.html#genesisconfig)
+struct in order to initialize Storage Items in the genesis block of your blockchain.
+
+// TODO
+
+## Accessing Storage Items
+
+Blockchains that are built with Substrate expose a remote procedure call (RPC) server that can be
+used to query your blockchain's runtime storage. You can use software libraries like
+[Polkadot JS](https://polkadot.js.org/) to easily interact with the RPC server from your code and
+access storage items. The Polkadot JS team also maintains
+[the Polkadot Apps UI](https://polkadot.js.org/apps), which is a fully-featured web app for
+interacting with Substrate-based blockchains, including querying storage. Refer to
+[the advanced storage documentation](../advanced/storage) to learn more about how Substrate uses a
+key-value database to implement the different kinds of Storage Items and how to query this database
+directly by way of the RPC server.
+
+## Child Storage Tries
 
 TODO
 
 ## Storage Cache
 
 TODO
 
-## Child Storage Tries
+## Best Practices
```

Comment:
Let's add:

- Verify first, write last
- Store only consensus critical info on chain (use example of submitting preimage)
- Set limits to the length of vectors that users can affect